src/factory/Layer.js

```diff
@@ -86,6 +106,61 @@ export const LayerFactory = {
     return layer;
   },
 
+  /**
+   * Returns an OpenLayers WFS layer instance due to given config.
+   *
+   * @param  {Object} lConf  Layer config object
+   * @return {ol.layer.Tile} OL WFS layer instance
+   */
+  createWfsLayer: function (lConf) {
+    // set a default projection if not set in config
+    if (!lConf.projection) {
+      lConf.projection = 'EPSG:3857';
+    }
+    // set a default WFS version if not set in config
+    if (!lConf.version) {
+      lConf.version = '1.1.0';
+    }
+    if (!lConf.format) {
+      lConf = 'GML3';
```

Comment:
In my experience (GeoExt, Heron, but OpenLayers 2) if nothing is specified for WFS and GML version, the best chance of interworking is still WFS 1.1.0 with GML2. GML3 is/was often problematic. But things may have changed...

Reply:
Plus: is meant at L125: `lConf.format = 'GML3'` else `lConf` is overwritten?

Reply:
WFS 1.1.0 with its default outputFormat GML3 works quite well. See https://docs.geoserver.org/stable/en/user/services/wfs/basics.html

> Plus: is meant at L125: lConf.format = 'GML3' else lConf is overwritten?

:see_no_evil: oh yes, good catch.